### PR TITLE
Releasenotes, spec change for 0.8.0-alpha

### PR DESCRIPTION
**Pull request by christian-kreuzberger-dtx**
This PR sets the spec version and preliminary release notes for Keptn 0.8.0-alpha and spec 0.2.0-alpha.

### CODE REVIEW

File: releasenotes/releasenotes_V0.8.0.alpha.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Release Notes 0.8.0.alpha
```
Comment (johannes-b) [inline releasenotes/releasenotes_V0.8.0.alpha.md:1]: Please change to: `0.8.0-alpha`

File: releasenotes/releasenotes_V0.8.0.alpha.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+Please note: This is an alpha release and should **not** be used for production workloads. 
+You should expect breaking changes, undocumented features and bugs.
+
+**The key announcements of Keptn 0.8.0.alpha**:
```
Comment (johannes-b) [inline releasenotes/releasenotes_V0.8.0.alpha.md:6]: 0.8.0-alpha

File: releasenotes/releasenotes_V0.8.0.alpha.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+
+## Keptn Specification
+
+Implemented **Keptn spec** version: [0.2.0.alpha](https://github.com/keptn/spec/tree/0.2.0.alpha)
```
Comment (johannes-b) [inline releasenotes/releasenotes_V0.8.0.alpha.md:11]: 0.2.0-alpha

File: releasenotes/releasenotes_V0.8.0.alpha.md (hunk 1 above)
Comment (christian-kreuzberger-dtx) [reply releasenotes/releasenotes_V0.8.0.alpha.md]: Fixed

File: releasenotes/releasenotes_V0.8.0.alpha.md (hunk 2 above)
Comment (christian-kreuzberger-dtx) [reply releasenotes/releasenotes_V0.8.0.alpha.md]: Fixed

**Review by johannes-b — APPROVED** [review]
lgtm

**Review by bacherfl — APPROVED** [review]

**Pull request merged**